python/core/is-your-python-healthy/python-debugger-ii.md

```diff
@@ -93,22 +93,16 @@ Finally, whenever you want to leave `pdb` console, type `quit` or `exit`.
 ---
 ## Practice
 
-What is the error this snippet will throw?
-```py
-name = input('What\'s your name?\n')
-def interaction()
-  print(name)
-  my_name = 'Seba'
-  if name is my_name:
-    print("We have the same name!")
-```
-???
+Match the command to what it does.

```

Comment:
don't think we need this phrase here as the users can understand what they have to do
```suggestion
```